### ISSUE TITLE
bug:

### ISSUE THREAD

**Issue by famewolf** [vikdevelop/SaveDesktop#339]
### Linux distribution

Garuda Dragonized KDE

### Desktop Environment

KDE Plasma

### Version of the SaveDesktop app

3.4.1-native

### Package type

Other

### Bug description

I ran the program and saved my settings but the resulting tar.gz appears to only be 113 bytes.   I've attached it.
[KDE-DE-Config.sd.tar.gz](https://github.com/user-attachments/files/17611678/KDE-DE-Config.sd.tar.gz)

Is it possibly not handling Plasma 6?

**Comment by vikdevelop** [comment]
Hello, try to run the SaveDesktop app in terminal (Ctrl+Alt+T) using the `savedesktop` command, save the current configuration, and then paste the output here.

**Comment by famewolf** [comment]
This is what showed up:

***@***.*** in ~ as 🧙 took 1m3s
╰─λ savedesktop

(savedesktop:2295731): Gtk-WARNING **: 14:52:14.406: Unknown key gtk-modules in /home/famewolf/.config/gtk-4.0/settings.
ini

(savedesktop:2295731): Adwaita-WARNING **: 14:52:14.448: Using GtkSettings:gtk-application-prefer-dark-theme with libadw
aita is unsupported. Please use AdwStyleManager:color-scheme instead.

(savedesktop:2295731): Adwaita-CRITICAL **: 14:52:30.806: adw_header_bar_pack_end: assertion 'gtk_widget_get_parent (chi
ld) == NULL' failed
python3: can't open file '/home/famewolf/.local/share/savedesktop/src/config.py': [Errno 2] No such file or directory
tar: .: file changed as we read it
Configuration saved successfully.

**Comment by vikdevelop** [comment]
From the output, it looks like the script for running the saving configuration process isn't starting because the directory doesn't exist. 
Do you have SaveDesktop installed from the AUR repository when the native version from this Github repository does not have this error?

**Comment by famewolf** [comment]
I installed from the aur and it compiled it.  I try to avoid flatpaks but can give it a go.

**Comment by vikdevelop** [comment]
In that case, you can report this issue to the real [AUR package](https://aur.archlinux.org/packages/savedesktop) maintainer, as I am not maintaining this package.

**Comment by famewolf** [comment]
Sorry for the confusion.

**Comment by famewolf** [comment]
> AUR maintainer here. I've addressed the issue with 3.5-2.

Just tried 3.5.2.  When saving the file it still ends up being 115 bytes instead of the file it should be.

**Comment by yochananmarqos** [comment]
@vikdevelop I can reproduce the above. Why didn't my change in [3.5-2](https://aur.archlinux.org/cgit/aur.git/commit/?h=savedesktop&id=dd921e31cd86869add0f8dd64b7bcf2de07f04d5) resolve the issue?

````
python3: can't open file '/home/yochanan/.local/share/savedesktop/src/config.py': [Errno 2] No such file or directory
creating and moving the configuration archive to the user-defined directory
tar: .: file changed as we read it
Configuration saved successfully.
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
```

**Comment by vikdevelop** [comment]
> @vikdevelop I can reproduce the above. Why didn't my change in [3.5-2](https://aur.archlinux.org/cgit/aur.git/commit/?h=savedesktop&id=dd921e31cd86869add0f8dd64b7bcf2de07f04d5) resolve the issue?
> 
> ```
> python3: can't open file '/home/yochanan/.local/share/savedesktop/src/config.py': [Errno 2] No such file or directory
> creating and moving the configuration archive to the user-defined directory
> tar: .: file changed as we read it
> Configuration saved successfully.
> shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
> ```
> ```

Just replace the 
```
sed -i 's|{home}/.local/share/savedesktop/translations|/usr/share/savedesktop/translations|g' \
    src/localization.py
sed -i 's|{Path.home()}/.local/share/savedesktop|/usr/share/savedesktop|g' "$pkgname"
``` 
this 
```
sed -i -e 's|{home}/.local/share/savedesktop|/usr/share/savedesktop|g; s|{Path.home()}/.local/share/savedesktop|/usr/share/savedesktop|g' src/localization.py savedesktop
``` 
in the PKGBUILD.

**Comment by yochananmarqos** [comment]
@vikdevelop That works, thanks. I thought my lines were already doing that. :laughing: 

@famewolf Fixed with 3.5-3.
